### PR TITLE
Fix: missing event.origin and event.environment tags for non-native events on iOS

### PR DESCRIPTION
**Pull request by marandaneto**
## :scroll: Description
Fix: missing event.origin and event.environment tags for non-native events on iOS #369

## :bulb: Motivation and Context
similar to https://github.com/getsentry/sentry-cordova/pull/204

## :green_heart: How did you test it?
triggering an event on flutter and ios

## :pencil: Checklist

- [X] I reviewed submitted code
- [X] I added tests to verify changes
- [ ] I updated the docs if needed
- [X] All tests passing
- [X] No breaking changes

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -5,6 +5,7 @@
 * Fix: Enable breadcrumb recording mechanism based on platform (#366)
 * Feat: Send default PII options (#360)
 * Bump: sentry-cocoa to v6.2.1 (#360)
+* Fix: missing event.origin and event.environment tags for non-native events on iOS
```
Comment (marandaneto) [inline CHANGELOG.md]:
```suggestion
* Fix: missing event.origin and event.environment tags for non-native events on iOS (#369)
```

File: flutter/lib/src/default_integrations.dart [hunk 2]
```diff
@@ -157,6 +157,15 @@ class LoadContextsIntegration extends Integration<SentryFlutterOptions> {
             sdk.addPackage(package['sdk_name']!, package['version']!);
             event = event.copyWith(sdk: sdk);
           }
+
+          // on iOS, captureEnvelope does not call the beforeSend callback,
+          // hence we need to add these tags here.
+          if (event.sdk?.name == 'sentry.dart.flutter') {
```
Comment (ueman) [inline flutter/lib/src/default_integrations.dart:163]: Does this interfer with Androids native integration?

Comment (marandaneto) [reply flutter/lib/src/default_integrations.dart:163]: no, this integration is iOS only

**Review by ueman — APPROVED** [review]
LGTM

**Pull request merged**